File: Niveau 2/4 – Fonctions/1 - Code secret deux fois.py
def pwIsSecret(secret, notFirst):
    print("Entrez le code :")
    password = input()
    if password == secret:
        if not(notFirst):
            print("Encore une fois.")
            return True
        else: 
            print("Bravo.")
            return True
    return False

secret = "4242"
b = False
c = False
while not(b) or not(c):
    if b:
        c = pwIsSecret(secret, b)
    else:
        b = pwIsSecret(secret, b)

File: Niveau 2/4 – Fonctions/1 - Code secret deux fois.cpp
#include <iostream>
using namespace std;

void mdp()
{
  int password = 0;
  while (password != 4242)
  {
    cout << "Entrez le code :" << endl;
    cin >> password;
  }
}

int main()
{
  mdp();
  cout << "Encore une fois." << endl;
  mdp();
  cout << "Bravo.";
}
